### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dA505819**
Added Semi-colons and a comma wherever necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,70 +31,70 @@ Statuspage's API wrapper
...
-Provides rapidjson support with parser and renderer
+Provides rapidjson support with parser and renderer:
 
   * [https://github.com/allisson/django-rest-framework-rapidjson](https://github.com/allisson/django-rest-framework-rapidjson)
 
 
 ### python-simple-rest-client
 
-Simple REST client for python 3.5+
+Simple REST client for python 3.5+ :-
```
Comment (raphaelfruneaux) [inline README.md]: I guess that hyphen could be removed, couldn't?
```suggestion
Simple REST client for python 3.5+ :
```

**Review by raphaelfruneaux — CHANGES_REQUESTED** [review]

**Comment by dA505819** [comment]
Done!
Thankyou for telling.

File: README.md (hunk 1 above)
Comment (dA505819) [reply README.md]: Done

**Review by doughss — APPROVED** [review]

**Comment by doughss** [comment]
Thanks @dA505819 :smile:

**Review by raphaelfruneaux — APPROVED** [review]
Thanks a lot @dA505819! 🎉

**Comment by dA505819** [comment]
Happy to help!

**Pull request merged**